README.md

```diff
@@ -82,6 +82,7 @@ You may [restart the session or reboot the watch](https://asteroidos.org/wiki/us
 | ![thumbnail](.thumbnails/prominent-seconds-round.webp) |![thumbnail](.thumbnails/prominent-seconds.jpg) | [prominent-seconds](prominent-seconds/usr/share/asteroid-launcher/watchfaces/prominent-seconds.qml) | [eLtMosen](https://github.com/eLtMosen) |
 | ![thumbnail](.thumbnails/pulsedot-round.webp) | ![thumbnail](.thumbnails/pulsedot.jpg) | [pulsedot](pulsedot/usr/share/asteroid-launcher/watchfaces/pulsedot.qml) | [eLtMosen](https://github.com/eLtMosen) |
 | ![thumbnail](.thumbnails/rainbow-uprising-round.webp) |![thumbnail](.thumbnails/rainbow-uprising.jpg) | [rainbow-uprising](rainbow-uprising/usr/share/asteroid-launcher/watchfaces/rainbow-uprising.qml) | [eLtMosen](https://github.com/eLtMosen) |
+| ![thumbnail](.thumbnails/resistance-is-futile-round.webp) |![thumbnail](.thumbnails/resistance-is-futile.jpg) | [resistance-is-futile](resistance-is-futile/usr/share/asteroid-launcher/watchfaces/resistance-is-futile.qml) | [beroset](https://github.com/beroset) |
```

Comment:
Both images are supplid as webp but the link to the non round one says jpg. Happens to me usually when i copy a line and alter it for a new WF. Since the legacy non-round images are still supplied as jpg to minimize the size of the git repo overall.

Reply:
good catch!